script.js

```diff
@@ -271,9 +271,41 @@ const loadWorks = (data) => {
 
 window.addEventListener('DOMContentLoaded', () => {
   loadWorks(data);
+  const formData = JSON.parse(window.localStorage.getItem('formDatas'));
+  if (formData) {
+    const email = document.querySelector('.email-area');
+    const message = document.querySelector('.text-area');
+    const fullName = document.querySelector('.name-area');
+    email.value = formData.email;
+    message.value = formData.message;
+    fullName.value = formData.fullName;
+  }
 
   const loadModalEle = document.querySelector('#loadModal');
   loadModalEle.addEventListener('click', () => {
     loadModal(loadModalEle.value);
   });
 });
+
+const form = document.querySelector('form');
+const alertMsg = document.querySelector('small');
+
+form.addEventListener('submit', (event) => {
```

Comment:
- [ ] Good job so far 👏 👏  I would like to point out that the form input should not be saved into the local storage when you submit the form, **rather anytime one of the form input values CHANGES** you can see that It is what is in the project requirements 👇 👇 
![image](https://user-images.githubusercontent.com/90222110/197286754-5da49a77-8315-4e92-8051-23c984b418a4.png)

That means the event should be `change` instead of `submit`. Please try to work out your codes so that the requirements are met 😃 👍 

**NOTE** The form data should be saved in the local storage so that even when I refresh the page without submitting the form, they should be autocompleted 😉

Reply:
implemented as pointed out. Thanks for the feedback